woocommerce/woocommerce-android
# [Woo POS] Tracking preparaion

Closes: #11517


### Description

The PR:
* Decouple current tracking logic from the POS, but reusing that
* Adds `woocommerceandroid_pos_` to POS events
* Makes tracking more safe and enforces us to do it without blocking the UI

### Testing instructions

* Use the app and notice that there are no changes in tracking for the current screens
* Use the POS screen and navigate to the checkout screen -> notice test event tracked with new prefix

Use tracks live view to do so 

### Images/gif
